templates/job-extend-preview.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
I am not sure if adding a new template is the right approach here. Have you considered updating the existing 'preview' one and remove and rename the existing buttons?

Reply:
Hmmm, I went back and forth between using a new template and using the 'preview' template. I didn't want to add too much conditional logic in that template since there is none at all currently. I do however think it is better to use the same template now because simple paid listings kinda depends on it.
I'm working on a fix for this now...

Reply:
Fixed in https://github.com/Automattic/WP-Job-Manager/pull/2422/commits/bb49c7a4b68b4ffab516267d914797a70853b34e

Reply:
Yep, this was the main problem with it, that users that have done customizations must define a second template while simply updating the existing one should be easier.